dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitorTest.kt

```diff
@@ -981,13 +987,81 @@ internal class DatadogRumMonitorTest {
 
     @Test
     fun `removes callback from handler on stopKeepAliveCallback`() {
+        // When
+        testedMonitor.stopKeepAliveCallback()
+
+        // Then
         // initial post
         verify(mockHandler).postDelayed(any(), any())
+        verify(mockHandler).removeCallbacks(same(testedMonitor.keepAliveRunnable))
```

Comment:
seems it should be `verifyNoMoreInteractions(mockHandler, mockWriter, mockScope)` as well here. also in the previous code `verify(mockHandler).postDelayed(any(), any())` comes before `stopKeepAliveCallback` (not sure why)

Reply:
The `postDelayed` is called in the `init` so normally it should come first. Not sure why was like that either.